Cargo.toml

```diff
@@ -12,10 +12,14 @@ repository = "https://github.com/snapview/tungstenite-rs"
 version = "0.13.0"
 edition = "2018"
 
+[package.metadata.docs.rs]
+features = ["native-tls"]
+
 [features]
-default = ["tls"]
-tls = ["native-tls"]
-tls-vendored = ["native-tls", "native-tls/vendored"]
+default = []
```

Comment:
I would suggest to enable TLS support by default. (and as discussed above, the default should lead to native TLS for now) I'm a big fan of sane defaults, and TLS enabled is arguable a very sane default in 2021.

People with special needs can still use `default-features = false`.

Reply:
I myself prefer the _tokio-style_ having nothing enabled by default and make it more of a _pick what you need_.

But I don't mind enabling `native-tls` as default, so I'll update this once the feature flag naming is resolved.